docs/asset-store/docs/08-01-gateway-mode-set-up.md

```diff
@@ -47,44 +53,51 @@ You can set Minio to the GCS Gateway mode during Kyma installation, or switch to
     ```bash
     gcloud iam service-accounts keys create $SECRET_FILE --iam-account=$SA_NAME@$PROJECT.iam.gserviceaccount.com
     ```
-6. Export the private key as the environment variable:
-    ```bash
-    export GCS_KEY_JSON=$(< ${SECRET_FILE} base64 | tr -d '\n')
-    ```
-7. Apply the following ConfigMap with an override onto a cluster or Minikube. Run:
+5. Export the private key as the environment variable:
     ```bash
-    cat <<EOF | kubectl apply -f -
-    apiVersion: v1
-    kind: Secret
-    metadata:
-      name: asset-store-overrides
-      namespace: kyma-installer
-      labels:
-        installer: overrides
-        component: assetstore
-        kyma-project.io/installation: ""
-    type: Opaque
-    data:
-      minio.gcsgateway.gcsKeyJson: "${GCS_KEY_JSON}"
-    ---
-    apiVersion: v1
-    kind: ConfigMap
-    metadata:
-      name: asset-store-overrides
-      namespace: kyma-installer
-      labels:
-        installer: overrides
-        component: assetstore
-        kyma-project.io/installation: ""
-    data:
-      minio.persistence.enabled: "false"
-      minio.gcsgateway.enabled: "true"
-      minio.defaultBucket.enabled: "false"
-      minio.gcsgateway.projectId: "${PROJECT}"
-      global.minioExternalEndpoint: "https://storage.googleapis.com"
-    EOF
+    export GCS_KEY_JSON=$(< "${SECRET_FILE}" base64 | tr -d '\n')
     ```
-8. Lable the `kyma-installtion` custom resource by running `kubectl label installation/kyma-installation action=install` to trigger Kyma installation.
 
+### ConfigMap
```

Comment:
it is not just a configmap, anyway I think that if it comes to steps, headings should describe action/step

ConfigMap = Configure Minio Gateway mode